lib/cb/utils/api.rb

```diff
@@ -29,9 +29,11 @@ def self.instance(headers: {})
         api
       end
 
-      def initialize(headers: {})
-        self.class.default_params developerkey: Cb.configuration.dev_key,
-                                  outputjson: Cb.configuration.use_json.to_s
+      def initialize(headers: {}, use_default_params: true)
+        if use_default_params
```

Comment:
This can be moved to its own method which would remove the CC complexity issue. Also, should this be tested (have a call with and without default params)?

```ruby
self.class.default_params add_default_params if use_default_params

def add_default_params
  { 
    developerkey: Cb.configuration.dev_key, 
    outputjson: Cb.configuration.use_json.to_s
  }
end
```